### PR TITLE
Update game commands for 2019 game

### PR DESCRIPTION
**Pull request by KernelPainc**
This pr updates many of the game commands to use the 2019 dates, and adds a new command `:game` that will give a countdown of the days before the game starts. This is my first time programming a discord bot, and as such I have no way of testing if these changes will work.

### CODE REVIEW

**Comment by KernelPainc** [comment]
I have no idea what just happened, I pressed "Update Branch"

**Comment by Ge0rg3** [comment]
Looks good to me.

File: bot/cogs/cyber.py [hunk 1]
```diff
@@ -181,15 +187,28 @@ def __init__(self, bot: Bot):
 
         await ctx.send(embed=embed)
 
+    @command()
+    async def game(self, ctx: Context):
+        # Get the current date
+        today = datetime.date.today()
+        rd = relativedelta(today, datetime.date(2019, 1, 15))
+        if today > rd:
```
Comment (AnonGuy) [inline bot/cogs/cyber.py]: image

File: bot/cogs/cyber.py [hunk 2]
```diff
@@ -181,15 +187,28 @@ def __init__(self, bot: Bot):
...
+            await ctx.send("Cyberstart Game has begun! Use :level base level to get info"
+                           "on specific challenges once we update the bot")
+            return
+        await ctx.send("Cyberstart Game begins on the 15th January 2019")
+        await ctx.send("Thats in {} months and {} days".format(rd.__dict__.months, rd.__dict__.days))
```
Comment (AnonGuy) [inline bot/cogs/cyber.py]: Why not `rd.months, rd.days`?

Comment (KernelPainc) [reply bot/cogs/cyber.py]: Changed.

File: bot/cogs/cyber.py [hunk 3]
```diff
@@ -181,15 +187,28 @@ def __init__(self, bot: Bot):
 
         await ctx.send(embed=embed)
 
+    @command()
+    async def game(self, ctx: Context):
+        # Get the current date
+        today = datetime.date.today()
+        rd = relativedelta(today, datetime.date(2019, 1, 15))
```
[the remark this answers is not in the document]
Comment (KernelPainc) [reply bot/cogs/cyber.py]: Changed.

**Comment by KernelPainc** [comment]
Thanks @AnonGuy!

**Pull request merged**